test/unstable-keyword/enum_test1.chpl

```diff
@@ -0,0 +1,14 @@
+enum foo {@unstable a, b, c};
+enum foo2 {@unstable a,};
+
+proc main() {
+    var trig = foo.a;   //triggered here because a in foo is what's marked
+    var trig2 = foo2.a; //triggered here
+
+    writeln(trig);  //not triggered here because unstable isn't attached to trig or trig2
+    writeln(trig2); //not triggered here
+
+    writeln(foo.a); //would be triggered because a is what's marked as unstable
+}
+
+main();
```

Comment:
Looks like this test doesn't have a .good file, so it'll probably fail in its current state

Reply:
I don't see this file removed or a .good file added for it yet